lib/mountain_view/component.rb

```diff
@@ -2,6 +2,10 @@ module MountainView
   class Component
     attr_reader :name
 
+    def self.stub_example
+      File.read(File.expand_path("../example.yml", __FILE__))
+    end
```

Comment:
What do you think about moving this to an instance method?
It's rarely called without instantiating a `MountainView::Component`.

Also, that way we could eventually make the file an `erb` and pass variables if needed. (may be useful if we end up adding a presenter class with the properties defined in the classes that inherit from it)